### PR TITLE
Hotfix polyline width

### PR DESCRIPTION
**Pull request by dydkask**
width add

### CODE REVIEW

File: packages/atlas/lib/src/polyline.dart [hunk 1]
```diff
@@ -13,10 +13,13 @@ class Polyline {
   /// The `Color` of the line
   final Color color;
 
+  final int width;
```
Comment (dakrawczyk) [inline packages/atlas/lib/src/polyline.dart:20]: Please add documentation describing this new property. Thanks

Comment (dydkask) [reply packages/atlas/lib/src/polyline.dart:20]: ok i add documentation

**Review by dakrawczyk — DISMISSED** [review]

File: packages/atlas/lib/src/polyline.dart [hunk 2]
```diff
@@ -13,10 +13,17 @@ class Polyline {
...
+  ///
+  /// The width is constant and independent of the camera's zoom level.
+  /// The default value is 10.
+  final int width;
+
   const Polyline({
     @required this.id,
     @required this.points,
     this.color = Colors.black,
+    this.width = 10,
```
Comment (felangel) [inline packages/atlas/lib/src/polyline.dart:26]: can we add a test for this?

**Review by felangel — CHANGES_REQUESTED** [review]
tests for default width

File: packages/atlas/lib/src/polyline.dart (hunk 2 above)
Comment (dydkask) [reply packages/atlas/lib/src/polyline.dart:26]: i can add test.

**Review by felangel — APPROVED** [review]

**Review by dakrawczyk — APPROVED** [review]

**Review by noahmateen — APPROVED** [review]

**Pull request merged**